docs/_data/home/heading.yml

```diff
@@ -0,0 +1,7 @@
+heading: Send NHS App messages, emails, tests and letters to patients and the public
+description: |
+  You can use NHS Notify if you work in NHS England and support direct care
+image: landing-main-image.svg
+image_alt: |
+  A graphic representation of NHS Notify&#39;s message channels:
```

Comment:
Interesting, is there a way to automatically HTML encode the text so that content designers can alter this without any HTML knowledge?

Reply:
Hm... maybe. There seem to be some Jekyll Liquid functions to escape XML. Which might be useful to use -> https://jekyllrb.com/docs/liquid/filters/

I'll have a play.

Reply:
Worth doing if it's easy, but not worth a lot of effort.

Reply:
Had a bit of play I couldn't see anything obvious. So I've dropped it.

Reply:
would say that need to do the docs/_includes/components/image.html where the {{ include.alt }} happens, will need the encode there